### PR TITLE
Add some rules

### PR DESCRIPTION
**Pull request by odinuge**
Hopefully it werkz

### CODE REVIEW

File: brus/liste/api/views.py [hunk 1]
```diff
@@ -1,14 +1,39 @@
...
+            current_balance = person.balance
+            for txn in person.transactions.filter(
+                date__gt=timezone.now() - timezone.timedelta(days=3)
+            ).reverse():
+                current_balance -= txn.value
+                if current_balance >= 0:
+                    break
+
+            # If user has not had a positive balance in 3 days return err
+            if current_balance < 0:
```
Comment (liseph) [inline brus/liste/api/views.py:23]: So you are not allowed to buy if you have had a negative balance for 3 consecutive days?

Comment (odinuge) [reply brus/liste/api/views.py:23]: Yeah. Can use this as a warning instead of error tho.

**Review by liseph — APPROVED** [review]
lgtm :beer:

**Review by SmithPeder — APPROVED** [review]
Looks good, but yeah

> Odin: "Yeah. Can use this as a warning instead of error tho."

maybe this is a good "first implementation" of this feature

**Pull request merged**